src/main/java/uk/gov/companieshouse/reconciliation/insolvency/InsolvencyCompanyNumberCompareMongoDBOracle.java

```diff
@@ -7,24 +7,24 @@
 import org.springframework.stereotype.Component;
 
 /**
- * Trigger a comparison between companies with insolvency cases on Oracle and MongoDB.
+ * Trigger a comparison between companies with insolvency cases on MongoDB and Oracle.
  */
 @Component
-public class InsolvencyCompanyNumberCompare extends RouteBuilder {
+public class InsolvencyCompanyNumberCompareMongoDBOracle extends RouteBuilder {
     @Override
     public void configure() throws Exception {
-        from("{{endpoint.insolvency.company_number.timer}}")
+        from("{{endpoint.insolvency.company_number.mongo_oracle.timer}}")
```

Comment:
Incorrect period separator: `company_number.mongo_oracle` should read `company_number_mongo_oracle`